src/components/VAudioTrack/VAudioTrack.vue

```diff
@@ -312,18 +314,47 @@ export default defineComponent({
     const play = () => {
       // delay initializing the local audio element until playback is requested
       if (!localAudio) initLocalAudio()
-      localAudio?.play()
+
+      try {
+        localAudio?.play()
```

Comment:
We need to handle the [`Promise`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/play#return_value) returned by the `play()` function, so catching the exception should look more like this:
```suggestion
        localAudio?.play().catch((err) => {
        if (err.name === 'AbortError') {
          // The play() request was interrupted by a call to pause()
          // therefore we can safely ignore it.
          return
        }
        // Send issue to Sentry...
      })
```
This will solve #1600 for sure, but #1605 will need to show a message to the user so they are aware of why they can't play the track.